core/core.go

```diff
@@ -111,12 +111,12 @@ func evaluateWatchdog(lastcheckin time.Time, watchdog int) bool {
 }
 
 func findPeers(last time.Time, sandcatAgent *agent.Agent) bool {
-    minDiscoveryInterval := 300
-    diff := float64(time.Now().Sub(last).Seconds())
-    if diff >= float64(rand.Intn(120) + minDiscoveryInterval) {
-        sandcatAgent.DiscoverPeers()
-        return true
-    } else {
-        return false
-    }
-}
\ No newline at end of file
+	minDiscoveryInterval := 300
+	diff := float64(time.Now().Sub(last).Seconds())
+	if diff >= float64(rand.Intn(120)+minDiscoveryInterval) {
+		sandcatAgent.DiscoverPeers()
+		return true
+	} else {
+		return false
+	}
+}
```

Comment:
Were the whitespace changes here and above done to align with the rest of the code?  It looks like so based on what I see.